test/restore-object.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
+/* eslint-disable no-empty-function */
+
+var referee = require("@sinonjs/referee");
+var sinonSpy = require("../lib/sinon/spy");
+var sinonStub = require("../lib/sinon/stub");
+var restoreObject = require("../lib/sinon/restore-object.js");
+var assert = referee.assert;
+var refute = referee.refute;
+
+describe("restore-object", function() {
+    it("is defined", function() {
+        assert.isFunction(restoreObject);
+    });
+
+    it("works on empty objects", function() {
+        refute.exception(function() {
+            restoreObject({});
+        });
+    });
+
+    it("quietly ignores falsy input", function() {
+        refute.exception(function() {
+            restoreObject(false);
+            restoreObject(null);
+        });
+    });
+
+    it("works with no spies or stubs", function() {
+        refute.exception(function() {
```

Comment:
Should throw in my opinion. Again, not the result of calling `sinon.stub` being passed.